pkg/ddevapp/providerPantheon.go

```diff
@@ -2,11 +2,12 @@ package ddevapp
 
 import (
 	"io/ioutil"
-	"log"
 	"os"
 	"path/filepath"
 	"strings"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
There is a `log.Fatalf` in this file which I believe should use the new output logger?

Reply:
Fixed in b1367eed8d1ec8475863b7d7b751f1a783dba430